packages/jest-runtime/src/index.ts

```diff
@@ -1267,8 +1267,12 @@ export default class Runtime {
         res =>
           // TODO: will this work on windows? It might be better if `shouldInstrument` deals with it anyways
           res.url.startsWith(this._config.rootDir) &&
-          this._v8CoverageSources!.has(res.url) &&
-          shouldInstrument(res.url, this._coverageOptions, this._config),
+          shouldInstrument(
+            res.url,
+            this._coverageOptions,
+            this._config,
+            Array.from(this._v8CoverageSources!.keys()),
```

Comment:
The removed line was used to check if file was loaded from a test file. The keys of `this._v8CoverageSources` is the list of loaded files. Shall I add a comment, perhaps?

Reply:
>Shall I add a comment, perhaps?

yeah, I think so 👍